apps/server/src/modules/board/uc/card.uc.spec.ts

```diff
@@ -60,6 +60,10 @@ describe(CardUc.name, () => {
 		await module.close();
 	});
 
+	// afterEach(() => {
+	// 	jest.resetAllMocks();
+	// });
+
```

Comment:
this resets the mock in line 51 and causes the bug in "findCard" because there is no instance of BoardDoAuthorizable any more. Moved the reset in the setup but that moved the problem only for a while

Reply:
fixed